driver-core/src/main/com/mongodb/connection/ServerDescription.java

```diff
@@ -833,8 +833,8 @@ public boolean equals(final Object o) {
         }
 
         if (logicalSessionTimeoutMinutes != null
-                    ? !logicalSessionTimeoutMinutes.equals(that.logicalSessionTimeoutMinutes)
-                    : that.logicalSessionTimeoutMinutes != null) {
+                ? !logicalSessionTimeoutMinutes.equals(that.logicalSessionTimeoutMinutes)
```

Comment:
Nit: revert whitespace changes, esp. for otherwise unchanged files

Reply:
Reverted.